### DISCUSSION TITLE
Mode 4 speed limit

### DISCUSSION THREAD

**Discussion by rohne83** [Courseplay/courseplay#7282] [Q&A] [answered]
Hi, ich meine es gab dazu schonmal ein Issue, kann es aber nicht mehr finden. Wollte aber jetzt nicht gleich ein neues Issue dafür aufmachen.
Jedenfalls, wenn ich mit Mode 4 dünge läuft alles gut bis wieder befüllt werden muss. Zumindest beim Befüllen am Mode8-ULW ist danach wieder das alte Problem, dass der Düngerstreuer nur noch mit 10km/h weiter macht (Weg zurück zum letzten Feldarbeitskurs-Punkt und auch die Feldarbeit selbst).
War das noch offen oder gilt das schon als gelöst? Ist bei mir jetzt in der ganz frischen Version 6.4.02 aufgetreten, auf der Mittelsachsen mit Vanilla-Streuer, Bergmann-ULW (vom Modhub) und Traktoren aus dem Claas-DLC.

**Comment by Tensuko** [comment]
Eigentlich sollte das jetzt passen, aber scheinbar gibt es bei der ein oder anderen Sache doch noch Probleme.
Kannst du das noch mal genau so probieren, aber mit der ModHub Version, 6.4.0.0 ?
Dann können wir zumindestens eingrenzen, ob das wieder neu dazu gekommen ist, oder ob es in der Kombination ein generelles Problem ist.

**Reply by rohne83** [reply]
Scheint ein generelles Problem zu sein, tritt auch mit der ModHub-Version von CP auf. Ich werd mal schauen ob das nur die Kombination von Map und Fahrzeugen ist, oder ob das auch in anderen Varianten Probleme macht.
![fsScreen_2021_06_16_10_58_24](https://user-images.githubusercontent.com/71030138/122190412-15218f00-ce92-11eb-8be0-522836d68240.png)

**Reply by rohne83** [reply]
Also es tritt auch in anderen Kombinationen auf, auch zB auf der Felsbrunn und mit anderen Düngerstreuern. Aus irgendwelchen unerklärlichen Gründen scheint das mit dem Überladewagen für den Mode 8 zusammen zu hängen, bei mir wie gesagt Bergmann GTW 330 vom Modhub. Als ich spaßeshalber nochmal mit Annaburger ULW probiert hab (zumindest in der Konfiguration mit Saatgutschnecke und zwei Kammern), hab ich das Problem nicht beobachten können.
Dafür hat der Bergmann aber nicht das Problem aus #7279

**Reply by Tensuko** [reply]
7279 muss ich erst noch schauen, weil da haben wir eigentlich nichts dran geändert...

Heist das, es soll nur an bestimmten ÜLW liegen ? Kann ich mir auch kaum vorstellen, weil uns das eigentlich egal sein sollte, an was für einem ÜLW wir uns wieder auffüllen...

Alles schon etwas komisch...

**Reply by rohne83** [reply]
Tatsächlich komisch, aber wie es aussieht scheint es tatsächlich über irgendwelche Wechselwirkungen an den ULWs zu liegen. Grad mal mehrere Durchläufe gemacht, mit 1. dem Bergmann, 2. Stara (der kleine Vanilla-ULW mit ca 17000Liter), 3. Annaburger mit Trennwand und Saatgutschnecke, 4. Annaburger ganz normal, 5, zum sicher gehen nochmal der Stara. Beim Stara genau das gleiche wie beim Bergmann, der Düngerstreuer fährt nach dem Auffüllen nur noch mit 10km/h weiter. Wenn als Mode 8 dagegen der Annaburger verwendet wird, egal in welcher Konfig, gings im normalen Tempo weiter. Einstellungen für Mode 4 und Mode 8 ansonsten während der Tests komplett unverändert gelassen. Logfile im Anhang.
[log.txt](https://github.com/Courseplay/courseplay/files/6665372/log.txt)

**Reply by Tensuko** [reply]
Hast du Discord ? Wenn ja, schreibe mich mal bitte an: Tensuko #2300

**Reply by rohne83** [reply]
Hab ich noch nicht. Aber wollt ich mir die nächsten Tage mal einrichten.

**Reply by rohne83** [reply]
Kann dich da nicht hinzufügen. rohne83#9664

**Reply by rohne83** [reply]
Den Grund hab ich bisher weiter noch nicht herausgefunden. Habe neulich aber auch mal den Refill-Kurs mit Wartepunkt versehen und den ULW mit dem Saatgut manuell gesteuert. Interessanterweise trat auch da dieser Effekt mit den 10km/h auf.
Debug Channels 4, 8, 12 waren für mich leider auch nicht sonderlich hilfreich soweit. Ich hab mal das Logfile mit jenen Channels von einem Versuch noch mit normalem Mode8-ULW angehängt. Hat etwas gedauert bis überhaupt überladen werden konnte weil die Sämaschinen ja doch ziemlich kleine schwer zu treffende fillNodes haben.
[log refill ulw channel 4 8 12.txt](https://github.com/Courseplay/courseplay/files/6986174/log.refill.ulw.channel.4.8.12.txt)

Werd mir das ganze aber nochmal mit der 6.4.0.6 anschauen und berichten.

**Reply by rohne83** [reply]
Also in der 6.4.0.6 hatte ich zumindest mit dem Bergmann ULW jetzt keine Probleme mehr. Sähmaschine/Düngerstreuer hat nach dem Befüllen wieder auf die eingestellte Geschwindigkeit hochbeschleunigt.

**Reply by Tensuko** [reply]
Das klingt schon mal sehr gut, kannst du auch gerne im Discord schreiben.

**Comment by rohne83** [comment, marked as answer]
Thema mal zu machen. Problem ist behoben seit v6.4.0.6
